pontoon/base/templates/translate.html

```diff
@@ -125,9 +125,13 @@
             {% endif %}
 
             {% if user.is_authenticated() %}
+                {% if user.logged_via('fxa') and not user.logged_via('persona') %}
```

Comment:
@mathjazz to be more precise: logged_via() checks if user logged via given provider once. So, this will be true even if user logged in the past.